projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad.php

```diff
@@ -907,6 +915,20 @@ function wpcom_launchpad_is_paid_newsletter_enabled() {
 	return wpcom_launchpad_has_goal_paid_subscribers() && apply_filters( 'wpcom_launchpad_intent_paid_newsletter_enabled', false );
 }
 
+/**
+ * Checks if site has newsletter or write as site intent.
+ *
+ * @return bool True if the task list is enabled, false otherwise.
+ */
+function wpcom_launchpad_is_newsletter_or_write_intent() {
+	$intent = get_option( 'site_intent', false );
+	if ( 'newsletter' !== $intent || 'write' !== $intent ) {
```

Comment:
This condition will always be true. I think we may need `&&` here instead of `||`.
```suggestion
	if ( 'newsletter' !== $intent && 'write' !== $intent ) {
```